packages/@uppy/vue/src/DragDrop.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div ref="container" />
+</template>
```

Comment:
You might want to convert this into a render function so these files can just be plain `*.ts` instead of vue templates.

Reply:
Hmm. My main reservation on that is compatibility with Vue 2 and 3 considering that the way of writing render functions slightly changed. How do you suggest handling that? Is there a better way that I'm not thinking of?

Reply:
Ah, I hadn't actually seen that breaking change, that would definitely be a pain to make compatible between versions. All good.

Reply:
If you're interested, here's the docs for it: https://v3.vuejs.org/guide/migration/render-function-api.html#overview

It's not a major difference, but I think it may make compatibility harder. Although, if we're bundling the components, they're probably getting converted to one of those anyway

Reply:
The only way I've seen other libraries handle this is via vue-demi ([example](https://github.com/SeregPie/EChartsVue/blob/main/src/VueChart.js)). Bringing in an extra dependency and increasing the complexity with vue-demi probably isn't worth it just to add a one-line render function though. 

At the end of the day, the current solution is simple and works.

Overall this PR looks great,, looking forward to the release 👍

Reply:
I ultimately decided to do this, as it makes bundling and transpilation much simpler. I think that unless I include the template compiler or do some clever hack, it won't be cross compatible anyway, because it's just compiled to a render function anyway